src/components/full-text-search/result-mobile/index.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { useState } from "react";
+import { SearchResultRow } from "../hooks/use-manage-data";
+import { ResultBox } from "./result-box";
+import {
+  Stack,
+  Flex,
+  Text,
+  useBreakpointValue,
+  IconButton,
+} from "@chakra-ui/react";
+import {
+  ChevronLeft,
+  ChevronRight,
+  ChevronsLeft,
+  ChevronsRight,
+} from "react-feather";
+
+interface ResultMobileProps {
+  data: SearchResultRow[];
+  pageSize?: number;
+}
+
+interface PageState {
+  pageIndex: number;
+  pageSize: number;
+}
+
+type TableIconButtonProps = {
+  icon: any;
+  onClick:
+    | ((event: React.MouseEvent<HTMLElement, MouseEvent>) => void)
+    | undefined;
+  isDisabled: boolean;
+  colorScheme?: string;
+};
+export const TableIconButton: React.FC<TableIconButtonProps> = ({
+  icon,
+  onClick,
+  isDisabled,
+  children,
+  colorScheme,
+  ...rest
+}) => {
+  return (
+    <IconButton
+      size="sm"
+      {...rest}
```

Comment:
props die so übergeben werden sollten immer als letztes in die component gespreadet werden. sonst kannst du z.B. die props icon, borderWidth, onClick, etc. von außen nicht mehr überschreiben